enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/DefaultAnalyticsEventPublisher.java

```diff
@@ -100,17 +99,17 @@ public void handleWebsocketFrameRequest(WebSocketFrameRequest webSocketFrameRequ
 
     @Override
     public void init(Map<String, String> configuration) {
-        boolean elkEnabled = org.wso2.choreo.connect.enforcer.analytics.AnalyticsConstants.ELK_TYPE
-                .equalsIgnoreCase(ConfigHolder.getInstance().getConfig().getAnalyticsConfig().getType());
-        if (!elkEnabled && (StringUtils.isEmpty(configuration.get(AnalyticsConstants.AUTH_URL_CONFIG_KEY))
-                || StringUtils.isEmpty(AnalyticsConstants.AUTH_TOKEN_CONFIG_KEY))) {
-            logger.error(AnalyticsConstants.AUTH_URL_CONFIG_KEY + " and / or " +
-                    AnalyticsConstants.AUTH_TOKEN_CONFIG_KEY +
-                    "  are not provided. Hence assigning default values");
+        
+        String authUrlKey = configuration.get(AnalyticsConstants.AUTH_URL_CONFIG_KEY);
```

Comment:
This would print a warning log evenif the CC uses elk publishing. But this property is only required for choreo insights.

Reply:
If we dont set some value here, it wont initialise the reports. @slahirucd7 You checked this previously right?

Reply:
Previously there was a requirement to have both fields. `authURL` and `token` :
https://github.com/wso2/product-microgateway/blame/main/enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/analytics/DefaultAnalyticsEventPublisher.java#L109

Reply:
I will do a fix for this. If the reporter class is given or elk is running we can skip this warning logs.

Reply:
It looks like we no longer need auth key and endpoint when using ELK or event publisher. Hence I removed those properties and thrown an error when those are not provided for Choreo Analytics.